### PR TITLE
WAZO-2498: Add tenant_uuid to bus events headers

### PR DESCRIPTION
**Pull request by ja0x**
Depends-On: https://github.com/wazo-platform/xivo-bus/pull/49
Depends-On: https://github.com/wazo-platform/xivo-bus/pull/50

### CODE REVIEW

**Review by sduthil — CHANGES_REQUESTED** [review]
That's an impressive PR! Thank you very much for the work done Julien!

Some small details left to polish and we're good to go!

**Comment by sduthil** [comment]
Thank you so much for removing the `ArbitraryEvent`! I wanted to do that for a long time now :)

File: wazo_calld/plugins/conferences/notifier.py [hunk 1]
```diff
@@ -35,68 +35,125 @@ class ConferencesNotifier:
...
 
+    @staticmethod
+    def _build_headers(user_uuids=None, **kwargs):
+        headers = {}
+        for uuid in user_uuids or []:
+            headers[f'user_uuid:{uuid}'] = True
+
+        for kw in kwargs:
+            if kw:
+                headers[kw] = kwargs[kw]
```
[the remark this answers is not in the document]
Comment (ja0x) [reply wazo_calld/plugins/conferences/notifier.py:46]: That was a mistake, it was supposed to check the value of the attribute for nullness

File: wazo_calld/plugins/meetings/notifier.py [hunk 2]
```diff
@@ -26,42 +26,63 @@ class MeetingsNotifier:
     def __init__(self, bus_producer):
         self._bus_producer = bus_producer
 
+    @staticmethod
+    def _build_headers(user_uuids=None, **kwargs):
+        headers = {}
+        for key, value in kwargs.items():
+            headers[key] = value
```
[the remark this answers is not in the document]
Comment (ja0x) [reply wazo_calld/plugins/meetings/notifier.py:33]: Same here, nullness check for value was missing

**Review by sduthil — APPROVED** [review]

**Pull request merged**